index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head >
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <!-- Hello World, we are software developers! -->
+    <header id="header">
+      <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png
+      " alt="An image">
+      <nav id="nav-bar">
+        <button class="but" type="submit" name="submit">Element 1</button>
+        <button class="but" type="submit" name="submit">Element 2</button>
+        <button class="but" type="submit" name="submit">Element 3</button>
+      </nav>
+    </header>
+    <iframe id="video" width="420" height="315" src="https://www.youtube.com/watch?v=f9ip1iSAptQ">
+    </iframe>
```

Comment:
I think for better clarity, you should close the tag frame in the above line.

For example: 
<iframe id="id123" width="140" height="200" src="url"></iframe>